src/reports.cpp

```diff
@@ -961,20 +961,20 @@ static config unit_weapons(reports::context & rc, const unit *attacker, const ma
 
 static config unit_weapons(reports::context & rc, const unit *u)
 {
-	if (!u || u->attacks().empty()) return config();
-	map_location displayed_unit_hex = rc.screen().displayed_unit_hex();
-	config res;
-
-	//TODO enable after the string frezze is lifted
-	//const std::string attack_headline =
-	//		( u->attacks().size() > 1 ) ? N_("Attacks") : N_("Attack");
+	config res = config();
+	if ((u != NULL) && (!u->attacks().empty())) {
+		map_location displayed_unit_hex = rc.screen().displayed_unit_hex();
 
-	//add_text(res,  /*span_color(font::weapon_details_color)
-	//		+*/ attack_headline /*+ "</span>\n"*/ + '\n', "");
+		//TODO enable after the string frezze is lifted
```

Comment:
OK. I removed all the commenting and it compiles. Dunno what it should do or where to look. As a separate commit so it can be cherry-picked away if nobody wants it after all. Will push in a bit when VS finishes running WML checks and everything looks good.